sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/query.sql

```diff
@@ -0,0 +1,1461 @@
+        THEN 1
+      ELSE 0
+    END AS save_prompt_shown_cc,
+    CASE
+      WHEN event_category = 'credit_cards'
+        AND event_name = 'save_prompt_create'
+        THEN 1
+      ELSE 0
+    END AS save_prompt_create_cc,
+    CASE
+      WHEN event_category = 'credit_cards'
+        AND event_name = 'save_prompt_update'
+        THEN 1
+      ELSE 0
+    END AS save_prompt_update_cc,
+    --management
+    CASE
+      WHEN event_category = 'credit_cards'
+        AND event_name = 'management_add_tapped'
+        THEN 1
+      ELSE 0
+    END AS management_add_tapped_cc,
+    CASE
+      WHEN event_category = 'credit_cards'
+        AND event_name = 'management_card_tapped'
+        THEN 1
+      ELSE 0
+    END AS management_tapped_cc,
+    CASE
+      WHEN event_category = 'credit_cards'
+        AND event_name = 'modified'
+        THEN 1
+      ELSE 0
+    END AS modified_cc,
+    /*Addresses*/
+    --autofill
+    CASE
+      WHEN event_category = 'addresses'
+        AND event_name = 'form_detected'
+        THEN 1
+      ELSE 0
+    END AS form_detected_address,
+    CASE
+      WHEN event_category = 'addresses'
+        AND event_name = 'autofill_prompt_shown'
+        THEN 1
+      ELSE 0
+    END AS autofill_prompt_shown_address,
+    CASE
+      WHEN event_category = 'addresses'
+        AND event_name = 'autofill_prompt_expanded'
+        THEN 1
+      ELSE 0
+    END AS autofill_prompt_expanded_address,
+    CASE
+      WHEN event_category = 'addresses'
+        AND event_name = 'autofill_prompt_dismissed'
+        THEN 1
+      ELSE 0
+    END AS autofill_prompt_dismissed_address,
+    CASE
+      WHEN event_category = 'addresses'
+        AND event_name = 'autofilled'
+        THEN 1
+      ELSE 0
+    END AS autofilled_address,
+    --management
+    CASE
+      WHEN event_category = 'addresses'
+        AND event_name = 'management_add_tapped'
+        THEN 1
+      ELSE 0
+    END AS management_add_tapped_address,
+    CASE
+      WHEN event_category = 'addresses'
+        AND event_name = 'management_address_tapped'
+        THEN 1
+      ELSE 0
+    END AS management_tapped_address,
+    /*Bookmark*/
+    CASE
+      WHEN event_category = 'bookmarks_management'
+        AND event_name = 'copied'
+        THEN 1
+      ELSE 0
+    END AS bookmark_copied,
+    CASE
+      WHEN event_category = 'bookmarks_management'
+        AND event_name = 'edited'
+        THEN 1
+      ELSE 0
+    END AS bookmark_edited,
+    CASE
+      WHEN event_category = 'bookmarks_management'
+        AND event_name = 'folder_add'
+        THEN 1
+      ELSE 0
+    END AS bookmark_folder_add,
+    CASE
+      WHEN event_category = 'bookmarks_management'
+        AND event_name = 'open'
+        THEN 1
+      ELSE 0
+    END AS bookmark_open,
+    CASE
+      WHEN event_category = 'bookmarks_management'
+        AND event_name = 'open_all_in_new_tabs'
+        THEN 1
+      ELSE 0
+    END AS bookmark_open_all_in_new_tabs,
+    CASE
```

Comment:
Could we make the SQL a bit more concise by using an `IF` or `COUNTIF` here instead of a `CASE`, from what I can see this is a simple true or else which seems appropriate for an IF, CASE is a bit more verbose and unless we have multiple possible outcomes it seems to add verbosity unnecessarily.